numpy/core/multiarray.py

```diff
@@ -761,6 +761,7 @@ def dot(a, b, out=None):
     tensordot : Sum products over arbitrary axes.
     einsum : Einstein summation convention.
     matmul : '@' operator as method with out parameter.
+    linalg.multi_dot : Chained dot product.
```

Comment:
I'm curious if we need the fully qualified name here and below. Anyone know?

Reply:
Good point, thanks! Looking at the missing link in the reference to `dot` in the docs of `multi_dot` I didn't even consider that it could work without the subpackage name. I've tried just now with `multi_dot` here instead of `linalg.multi_dot` and the link disappeared.

I'm not familiar with sphinx nor numpydoc to try and figure out where this lookup might happen.

Reply:
Thanks for checking.